test/unit/canvas_events.js

```diff
@@ -481,6 +483,39 @@
     });
   });
 
+  QUnit.test('Fabric mouseover, mouseout events fire for subTargets when subTargetCheck is enabled', function(assert){
+    var counterOver = 0, counterOut = 0, canvas = new fabric.Canvas();
+    function setSubTargetCheckRecursive(obj) {
+      if (obj._objects) {
+        obj._objects.forEach(setSubTargetCheckRecursive);
+      }
+      // TODO: maybe this property could've been baked into the JSON?
+      // not sure if loadFromJSON accounts for it tho
```

Comment:
so the JSON file always ever cared for the graphic restore. No funcionalities. This is not written in stone, but for now is out of scope. This same problem has to be revised again when i get in the new control interface.